pkgs/applications/networking/instant-messengers/discord-screenaudio/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, wrapQtAppsHook
+, cmake
+, pkg-config
+, qtbase
+, qtwebengine
+, qtwayland
+, pipewire
+, nix-update-script
+}:
+
+stdenv.mkDerivation rec {
+  pname = "discord-screenaudio";
+  version = "1.9.2";
+
+  src = fetchFromGitHub {
+    owner = "maltejur";
+    repo = "discord-screenaudio";
+    rev = "v${version}";
+    hash = "sha256-it7JSmiDz3k1j+qEZrrNhyAuoixiQuiEbXac7lbJmko=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+    cmake
+    pkg-config
+    qtbase
+    qtwebengine
```

Comment:
This should be in buildInputs if no binaries from it are used

Reply:
Referring to the two qt packages here? Is that the rule in general? I always thought that it was if a package isn't needed at runtime, it should be put in nativeBuildInputs.

Reply:
Build time tools are in nativeBuildInputs

Libs and packages to be added to the wrapper (if in use) should be in buildInputs 

Add a `strictDeps = true` to separate the dependencies better and not add buildInputs to the build PATH

Reply:
It's good to use it always, note that some ecosystem specific builders such as `buildRustPackage` and `buildPythonApplication` always enable it.

Check with 
```
$ nix eval ".#bash.strictDeps"
true
```
when in doubt.